test/pseudo-tty/test-tty-get-color-depth.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
```

Comment:
I would like that we keep using strict mode. It is something that I would like to switch over on the long term.

Reply:
@BridgeAR That lint rule is not correctly applied to `/pseudo-tty/`, so the linter complains.

Reply:
You will have to deactivate that rule. The rule should actually be updated. I will open an issue so it can be grabbed by someone.